addons/l10n_ar/views/report_invoice.xml

```diff
@@ -230,6 +230,10 @@
             </div>
         </div>
 
+        <xpath expr="//div[@id='right-elements']" position="attributes">
+            <attribute name="t-attf-class">#{'col-7' if report_type != 'html' else 'col-12 col-md-5'} ms-5 d-inline-block float-end</attribute>
+        </xpath>
+
         <!--  we remove the ml auto and also give more space to avoid multiple lines on tax detail -->
```

Comment:
I think your solution works but it's not the best. You simply increase the size of the column, but the issue seems to be the fact that there is a block of field (Invoiced date, Currency, Exchange rate) that is not placed at the right position. What I would suggest is, instead of adding a xpath, simply change these lines: https://github.com/odoo/odoo/pull/152682/files#diff-a47fda1ec827f6205b725feabd30a901f66b34350b4b1feca21d21bf8327107bR237-R243 
by 

`<xpath expr="//div[@id='payment_term']" position="before">`
            `    <div class="mb-4">`

This way, the table with the totals and the details of taxes can takes it's full width.

Reply:
Thanks for the review!